src/pages/conference/speakers/index.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import styled from 'styled-components'
+import ImageContentBlock from './_speakers_list'
+import Name1 from './images/name1.svg'
+import Name2 from './images/name2.svg'
+import Name3 from './images/name3.svg'
+import Name4 from './images/name4.svg'
+import Footer from 'components/layout/footer'
+import { Localize } from 'components/localization'
+
+const Layout = styled.div`
+    background: var(--color-white);
+`
+
+const data = [
+    {
+        id: 'name_01',
+        image: Name1,
+        title: <Localize translate_text="Name" />,
```

Comment:
```suggestion
type TSpeaker = {
    id: string,
    image: string,
    title : TString,
    content: TString
}
const data : TSpeaker[] = [{
    id: "name_01",
    image: Name1,
    title: "Name",
    content: "Who is the speaker? This is where you can write a bio to detail your speaker’s credentials and the unique perspective they will give to the conference."
    }
}]
```
we can't have this kinda structure now, filtering content and applying changes will be easier, can you please update it, thanks thanks :pray: 
> Note: Please check the next suggestion as well.